client/state/terms/test/selectors.js

```diff
@@ -65,6 +66,66 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'isRequestingTermsForQueryIgnoringPage()', () => {
+		it( 'should return false if no request exists', () => {
+			const requesting = isRequestingTermsForQueryIgnoringPage( {
+				terms: {
+					queryRequests: {}
+				}
+			}, 2916284, 'categories', {} );
```

Comment:
Noticed in #6224 that most tests are specifying `categories` as the taxonomy, when the proper name should be `category` (technically, there could be another taxonomy named `categories`, but I assume this refers to the one for posts). I updated the affected tests in #6224 (rebased commit).

https://public-api.wordpress.com/rest/v1.1/sites/en.blog.wordpress.com/post-types/post/taxonomies